File: program139.java
//template for problems on n numbers
import java.lang.*;
import java.util.*;

class ArrayX
{
    private int Arr[];
    
    public ArrayX(int iNo)
    {
        Arr=new int[iNo];
    }

    public void Accept()
    {
        Scanner sobj=new Scanner(System.in);
        System.out.println("ENter the values:");
        
        for(int i=0;i<Arr.length;i++)
        {
            Arr[i]=sobj.nextInt();
        }
    }

    public void Display()
    {
        System.out.println("Elements of the array are:");
        for(int i=0;i<Arr.length;i++)
        {
            System.out.println(Arr[i]);
        }
    }

    //add function here with logic
    
 
}

class program_________
{
    public static void main(String arg[])
    {
        Scanner sobj=new Scanner(System.in);
        int iLength=0,iRet=0;

        System.out.println("Enter the value for constructor:");
        iLength=sobj.nextInt();

        ArrayX obj=new ArrayX(iLength);

        obj.Accept();
        obj.Display();

        iRet=obj.______();

        obj=null;

    }
}

File: program139.cpp
//addition of n numbers in oop
#include<iostream>
using namespace std;

class ArrayX
{
    private:
    int *Arr;
    int iSize;

    public:
    ArrayX(int iValue)
    {
        this->iSize=iValue;
        Arr=new int[iSize];  //resource

    }
    ~ArrayX()             //destructor
    {
        delete []Arr;   //Deallocate the resource
    }

    void Accept()  //void Accept(ArrayX *this)
    {
        int i=0;
        cout<<"Enter the elements od array"<<endl;
        
   for(int i=0;i<iSize;i++)
     {
       cin>>Arr[i];
     }
    }

    void Display()
    {
        cout<<"Elements of the array are"<<endl;
        for(int i=0;i<iSize;i++)
        {
            cout<<Arr[i]<<endl;
        }
    }

 //Add function here with logic
};

int main()
{
    int iRet=0;

    ArrayX obj1(5);

    obj1.Accept();  //Accpet(&obj)
    obj1.Display();

    iRet=obj1.__________()


    return 0;
}